Consul/Health.cs

```diff
@@ -291,16 +291,7 @@ public Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, boo
         /// <returns>A query result containing the service members matching the provided service ID, tag, and health status, or a query result with a null response if no service members matched the filters provided</returns>
         public Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, bool passingOnly, QueryOptions q, CancellationToken ct = default(CancellationToken))
         {
-            var req = _client.Get<ServiceEntry[]>(string.Format("/v1/health/service/{0}", service), q);
```

Comment:
This was unnecessary code duplication.

Reply:
https://github.com/G-Research/consuldotnet/blob/6d9b27d31b3724282a2f607d90dc05581e4bc938/Consul/Health.cs#L307-L319

I guess you can refactor this as well.